tests/test_tree_stats.py

```diff
@@ -1086,12 +1119,17 @@ def test_case_2(self):
         branch_true_mean_diversity = (
             0 + branch_true_diversity_02 + branch_true_diversity_01
             + branch_true_diversity_12) / 4
-        # Y(0;1, 2)
-        branch_true_Y = 0.2*4 + 0.2*(4+2) + 0.2*4 + 0.2*2 + 0.2*(5+1)
+        # y(0;1, 2) - 0.5 (y(1;0,2) + y(2;0,1)
+        # assume branch lengths are same on alternative topologies and distances
+        # are measured to UNROOTED mrca
+        branch_true_Y = 0.2*4 + 0.2*(4+2) + 0.2*4 + 0.2*2 + 0.2*(5+1) - 0.5 * (
+            (0.2*4 + 0.2*2 + 0.2*4 + 0.2*2 + 0.2*4) +
+            (0.2*(5+1) + 0.2*2 + 0.2*(5+1) + 0.1*(5+3) + 0.1*(4+2) + 0.2*4)
+        )
 
         # site stats
-        # Y(0;1, 2)
-        site_true_Y = 1
+        # Y(0;1, 2) = 0|12 - 0.5 * (1|02 + 2|01)
+        site_true_Y = 1 - 0.5 * (1 + 1)
```

Comment:
NB: I'm sure this is true by symmetry but I can't see it easily from the trees drawn above. Perhaps looking closely at the sites table would help (it did for recurrent muts example)